packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -399,17 +399,61 @@ class WebpackCLI {
             },
         ];
 
+        const normalizeType = (type, enumValues = []) => {
+            if (type === 'enum') {
+                const typeofValues = [];
+                enumValues.forEach((value) => {
+                    if (!typeofValues.includes(typeof value)) {
+                        typeofValues.push(typeof value);
+                    }
+                });
+                if (typeofValues.length === 1) {
+                    return normalizeType(typeofValues[0]);
+                } else {
+                    const types = typeofValues.map((type) => normalizeType(type));
+                    return types;
+                }
```

Comment:
Please avoid multiple loops, just return `['Number', 'String', 'Boolean']` here for `enum`